README.md

```diff
@@ -154,6 +154,10 @@ git push heroku master
 
 * This also runs all of the build and deployment commands, so it may take a moment, but once it's complete, the live link should be updated.
 
+## Running tests with Cypress.io
+
+Run `npx cypress open`
```

Comment:
Should probably just go ahead and put this behind an `npm` script.
something like
```
$ npm run test:e2e
```

Reply:
+++